src/data/battle_moves.h

```diff
@@ -5277,13 +5354,17 @@ const struct BattleMove gBattleMoves[MOVES_COUNT] =
 
     [MOVE_HOWL] =
     {
+        #if B_UPDATED_MOVE_DATA >= GEN_8
+            .target = MOVE_TARGET_SELECTED,
+        #else
+            .target = MOVE_TARGET_USER,
+        #endif
         .effect = EFFECT_ATTACK_UP,
         .power = 0,
         .type = TYPE_NORMAL,
         .accuracy = 0,
         .pp = 40,
         .secondaryEffectChance = 0,
-        .target = MOVE_TARGET_USER,
         .priority = 0,
         .flags = FLAG_SNATCH_AFFECTED | FLAG_SOUND,
         .split = SPLIT_STATUS,
```

Comment:
Howl shouldn't have a change in target (yet), but would need a modification in its battle script to loop through allies instead based on `B_UPDATED_MOVE_DATA`. So this change shouldn't be made to be clear.

In the future, we should take a look at which new target types we need to add instead of dumping them all into affecting user when it affects user + allies formally.